src/families/hedera/js-sync.ts

```diff
@@ -0,0 +1,49 @@
+import { Observable } from "rxjs";
+import { Account, SyncConfig } from "../../types";
+import { getOperationsForAccount } from "./api/mirror";
+import { getAccountBalance } from "./api/network";
+
+export default function sync(
```

Comment:
This implementation is not optimal as the full list of operations is replaced every time.
You can consider using `mergeOps`, see examples in other coin families.

There may be other concerns with not using the generic helpers, as already raised by @gre. We need to discuss and help you on that topic.

Reply:
To follow-up on this topic, I _think_ we are ok with having custom `sync` and `scanAccount` logics, to be confirmed by @gre.

But at minimum we need to make sure the performance is acceptable. The optimization issue that I mentioned above should be addressed.

Reply:
Digging a bit more, I'm wondering if there is a way to use the standard mechanism, by using `ChildAccount`... I need to think more about it.

Reply:
Never mind previous comment, probably a dead-end.

Reply:
👍🏼